platformio.ini

```diff
@@ -273,8 +273,9 @@ board_build.partitions = ${esp32.default_partitions}   ;; default partioning for
 ;;
 ;; please note that you can NOT update existing ESP32 installs with a "V4" build. Also updating by OTA will not work properly.
 ;; You need to completely erase your device (esptool erase_flash) first, then install the "V4" build from VSCode+platformio.
-platform = espressif32@ ~6.3.2
-platform_packages = platformio/framework-arduinoespressif32 @ 3.20009.0    ;; select arduino-esp32 v2.0.9 (arduino-esp32 2.0.10 thru 2.0.14 are buggy so avoid them)
+
+;; select arduino-esp32 v2.0.9 (arduino-esp32 2.0.10 thru 2.0.14 are buggy so avoid them)
+platform = https://github.com/tasmota/platform-espressif32/releases/download/2023.06.02/platform-espressif32.zip ;; Tasmota Arduino Core 2.0.9 with IPv6 support, based on IDF 4.4.4
```

Comment:
I was aiming to keep the same version of the Arduino Core as before, just swapping to Tasmota. 

Have I done this correctly? @Jason2866

Reply:
The platform (with removing the `platform_packages` entry) change is correct.

Reply:
But I wouldn't remove `platform_packages` completely. Just the entry.

Reply:
What should the correct value of `platform_packages` be then?

Reply:
Just leave it empy. As following
`platform_packages =`

Reply:
What impact would that have?